src/components/Cards/ImageCard.js

```diff
@@ -1,31 +1,51 @@
 import React, { PropTypes } from 'react';
-import { Card } from '../UI';
+import { Card, CardMedia, CardTitle, CardText } from 'react-toolbox/lib/card';
 import styles from './ImageCard.css';
 
-export default class ImageCard extends React.Component {
-
-  render() {
-    const { onClick, onImageLoaded, image, text, description } = this.props;
-    return (
-      <Card onClick={onClick} className={styles.root}>
-        <img src={image} onLoad={onImageLoaded} />
-        <h2>{text}</h2>
-
-        {description ? <p>{description}</p> : null}
-      </Card>
-    );
-  }
-}
+const ImageCard = (
+  {
+    author,
```

Comment:
Yes, I agree. I think we need to introduce some kind of schema for entries that can be used for both edit form generation as well as list view item. Thoughts?